includes/wpmenucart-settings.php

```diff
@@ -655,7 +655,9 @@ public function select_element_callback( $args ) {
 		$disabled = (isset( $args['disabled'] )) ? ' disabled' : '';
 		
 		$html  = sprintf( '<select name="%1$s[%2$s]" id="%1$s[%2$s]"%3$s>', esc_attr( $menu ), esc_attr( $id ), esc_attr( $disabled ) );
-		$html .= sprintf( '<option value="%s"%s>%s</option>', '0', selected( $current, '0', false ), '' );
+		if ( $args['id'] == 'shop_plugin' ) {
```

Comment:
I think

```if ( 'shop_plugin' === $args['id'] )```

would be safer.

Reply:
Thanks @brualio, done here: [dffb3fa](https://github.com/wpovernight/wp-menu-cart/pull/50/commits/dffb3faf83648b1b6383084d318be744b3d48285)